packages/e2e-test-utils/src/keyboard-mode.js

```diff
@@ -1,12 +1 @@
-/**
- * Triggers edit mode if not already active.
- *
- * @return {Promise} Promise resolving after enabling the keyboard edit mode.
- */
-export async function disableNavigationMode() {
-	const focusedElement = await page.$( ':focus' );
-	await page.click( '.editor-post-title' );
-	if ( focusedElement ) {
-		await focusedElement.focus();
-	}
-}
+
```

Comment:
Should this file have been removed? It remains as an empty, unreferenced file.

Reply:
😅